GlobalBehaviorandComponents/manageapps.py

```diff
@@ -87,12 +94,14 @@ def gbac_apps_delete():
 def gbac_apps_update():
     logger.debug("gbac_apps_update()")
     # user_info = get_userinfo()
-    # okta_admin = OktaAdmin(session[SESSION_INSTANCE_SETTINGS_KEY])
+    okta_admin = OktaAdmin(session[SESSION_INSTANCE_SETTINGS_KEY])
 
-    # app_name = request.args.get('appname')
-    # app_id = request.args.get('appid')
-    message = "Application Updated"
-    return redirect(url_for("gbac_manageapps_bp.gbac_apps", _external=True, _scheme=session[SESSION_INSTANCE_SETTINGS_KEY]["app_scheme"], message=message))
+    oidcclientid = request.args.get('oidcclientid')
+    oidcloginredirecturi = request.args.get('oidcloginredirecturi')
+    oidcapplabel = request.args.get('oidcapplabel')
+
+    okta_admin.update_web_application(app_label=oidcapplabel, redirect_uris=oidcloginredirecturi, app_id=oidcclientid)
+    return ""
```

Comment:
Just `return` remove `""`, unless you don't need it at all.

Reply:
Added, HTTP Response code.